### PR TITLE
browser(firefox): Browser.setScrollbarsHidden

### PR DESCRIPTION
**Pull request by JoelEinbinder**
For #6456

What do I do for firefox-stable?﻿

### CODE REVIEW

File: browser_patches/firefox/juggler/content/FrameTree.js [hunk 1]
```diff
@@ -102,12 +102,25 @@ class FrameTree {
   }
 
   _onDOMWindowCreated(window) {
+    if (!window.__plawrightScrollbarsStylesheet && this.scrollbarsHidden) {
```
Comment (dgozman) [inline browser_patches/firefox/juggler/content/FrameTree.js:105]: Why the flag? Do we come here multiple times? If so, let's maybe use a symbol instead?

File: browser_patches/firefox/juggler/content/FrameTree.js [hunk 2]
```diff
@@ -102,12 +102,25 @@ class FrameTree {
...
+      const styleSheetService = Cc["@mozilla.org/content/style-sheet-service;1"].getService(Components.interfaces.nsIStyleSheetService);
+      const ioService = Cc["@mozilla.org/network/io-service;1"].getService(Components.interfaces.nsIIOService);
+      const uri = ioService.newURI('chrome://juggler/content/content/hidden-scrollbars.css', null, null);
+      // styleSheetService.loadAndRegisterSheet(uri, styleSheetService.AGENT_SHEET);
```
Comment (dgozman) [inline browser_patches/firefox/juggler/content/FrameTree.js:109]: Commented code.

**Review by dgozman — APPROVED** [review]

File: browser_patches/firefox/juggler/content/FrameTree.js (hunk 2 above)
Comment (JoelEinbinder) [reply browser_patches/firefox/juggler/content/FrameTree.js]: done

File: browser_patches/firefox/juggler/content/FrameTree.js (hunk 1 above)
Comment (JoelEinbinder) [reply browser_patches/firefox/juggler/content/FrameTree.js]: Yeah, there is an event `juggler-dom-window-reused`.

**Pull request merged**